modules/stochastic_tools/src/interfaces/CovarianceInterface.C

```diff
@@ -8,6 +8,7 @@
 //* https://www.gnu.org/licenses/lgpl-2.1.html
 
 #include "CovarianceInterface.h"
+#include "CovarianceFunctionBase.h"
```

Comment:
Is this include needed?

Reply:
Yes, the warehouse-query is templated and has a dynamic cast which need the complete declaration.